app/assets/stylesheets/application.scss

```diff
@@ -1,6 +1,22 @@
-@import "bootstrap";
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+@import "bootstrap/scss/bootstrap";
 @import "font-awesome";
-@import "./authenticated";
+// @import "./authenticated";
```

Comment:
we can remove this

Reply:
Instructions had said to convert to scss and remove require tree, etc, but as long as it's working, happy to have it back! Since you put the old syntax back in there (*= vs @ import), should we also change the file extension back to css?

Reply:
Sorry @maebeale, i misunderstood what you had done. I was putting css in `listings.scss`, seeing them not take effect and tracked it down to that change. But i should've realized the appropriate fix would have been to explicitly `import "./listings"` instead of rollback your change.

Reply:
oh, no worries!!! :) but thanks for commenting that, @exbinary !
i think we do need to change the extension if going w require_tree tho ?

Reply:
Hmm, i didn't know that was a constraint. It seems to be getting imported doesn't it?

Reply:
Seems so! That makes sense, that the *= css syntax would be ok in scss file...so, yeah, prob fine to leave as scss.